### PR TITLE
QA fixes

### PR DESCRIPTION
**Pull request by abdullah-git1**

### CODE REVIEW

**Review by bryantaustin13 — APPROVED** [review]
LGTM

File: input/profiles/StructureDefinition-computable-measure-cqfm.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
             "key" : "cmp-5",
             "severity" : "error",
             "human" : "If type is specified at the root, it may not be specified on any group",
-            "expression" : "type.exists() implies group.select(extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-measureType').exists()).allFalse()"
+            "expression" : "type.exists() implies group.select(extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-measure').exists()).allFalse()"
```
Comment (brynrhodes) [inline input/profiles/StructureDefinition-computable-measure-cqfm.json:94]: Shouldn't this be `cqfm-type`, not `cqfm-measure`?

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Looks good other than the invariant fix seems incorrect to me?

File: input/profiles/StructureDefinition-computable-measure-cqfm.json (hunk 1 above)
Comment (abdullah-git1) [reply input/profiles/StructureDefinition-computable-measure-cqfm.json]: d'oh, invariant extension has been fixed

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**